cmd/user-agent/src/main.js

```diff
@@ -69,6 +69,8 @@ let defaultTrustBlocStartupOpts = {
     'log-level': 'debug',
     walletMediatorURL: '',
     credentialMediatorURL: '',
+    agentUsername: 'user-agent',
+    sdsServerURL: 'https://localhost:8072/encrypted-data-vaults'
```

Comment:
don't hardcode SDS URL in default opts, leave it as empty.

Reply:
@sudeshrshetty how is this different than the hard-coding in `defaultAriesStartupOpts` or the existing keys in `defaultTrustBlocStartupOpts`?

Part of this change also makes the SDS URL mandatory otherwise it will fail. Adding a toggle to choose local or remote storage is still a ways off into the future.

Reply:
Default ops won't work. Even walletMediatorURL is mandatory, we didn't set it. 

If we want edge-agent to work with default ops, then we need to initialize all required opts properly.

Reply:
@sudeshrshetty Note that at least some of the edge-agent functionality works without those other opts (but with the SDS URL). I tested registering wallet and DID creation and they seemed to work ok.

I can leave it as blank URL if you'd really prefer that - I just figured that it's a little more convenient so that you can do UI/JS work quickly. At least now with this change you'll get a more obvious error message.

Reply:
Yeah I am trying to understand what @sudeshrshetty is saying.... I see that `trustblocStartupOpts()` pulls the config from the backend... yet still uses `defaultTrustBlocStartupOpts` if `walletMediatorURL` is not present...

So `defaultTrustBlocstartupOpts` is the *fallback* and should definitely have *something* there. Config optional in the backend, mandatory in the frontend, from the looks of it.

Reply:
I agree @llorllale , then we should initialize it with default options for all mandatory options.

Reply:
@sudeshrshetty Should I create a follow-up story for that and leave the `https://localhost:8072/encrypted-data-vaults` arg as is?

Reply:
@DRK3 not needed for now